kubernetes/structure_service_spec.go

```diff
@@ -34,6 +34,22 @@ func flattenIPFamilies(in []v1.IPFamily) []interface{} {
 	return att
 }
 
+func flattenSessionAffinityConfigClientIP(in v1.ClientIPConfig) []interface{} {
+	a := make([]interface{}, 0, 0)
+	att := make(map[string]interface{})
+	att["timeout_seconds"] = in.TimeoutSeconds
+	a = append(a, att)
+	return a
+}
+
+func flattenSessionAffinityConfig(in v1.SessionAffinityConfig) []interface{} {
+	a := make([]interface{}, 0, 0)
```

Comment:
Because there's only one attribute here we could just write this as one return statement instead of using these `make` and `append` calls, like this:

```golang
return []interface{}{
	map[string]interface{}{
		"client_ip": flattenSessionAffinityConfigClientIP(*in.ClientIP),
	},
}
```

could also do the same in `flattenSessionAffinityConfigClientIP` above.

Reply:
Like this one, thank you!